README.md

```diff
@@ -40,7 +40,7 @@ or directly in pubspec.yaml
 
 ```yaml
 dev_dependencies:
-  lint_quido: 1.22.3
+  lint_quido: 1.24.5
```

Comment:
Let's keep the approach, where we use MINOR version component to match the Flutter version, and leaving PATCH for potential minor fixes to the package. In that case, it would be `1.24.0`

Reply:
Thanks, changed to bluesky... I mean 1.24.0